.ci/pytorch/win-test-helpers/build_pytorch.bat

```diff
@@ -76,6 +76,14 @@ set PATH=%CUDA_PATH%\bin;%CUDA_PATH%\libnvvp;%PATH%
 
 :cuda_build_end
 
+if not "%USE_XPU%"=="1" goto xpu_build_end
+
+set PYTHON_VERSION='3.10'
+set VS2022INSTALLDIR='C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools'
+set CONDA_ENV='xpu_venv'
```

Comment:
Same here, why we need this var for XPU? I think for XPU, we just need to set MSVC 2022 only, others can we reuse existed env?